test/c/test.cpp

```diff
@@ -236,6 +240,7 @@ BOOST_AUTO_TEST_CASE(ethash_check_difficulty_check) {
 
 BOOST_AUTO_TEST_CASE(test_ethash_dir_creation) {
     ethash_blockhash_t seedhash;
+	memset(&seedhash, 0, 32);
```

Comment:
hehe thanks for spotting them. Will fix them from emacs .. I have no idea what Visual studio does!